cmd/stakercli/transaction/transactions_test.go

```diff
@@ -223,19 +186,8 @@ func appRunCreatePhase1WithdrawalTx(r *rand.Rand, t *testing.T, app *cli.App, ar
 	return data
 }
 
-func randRange(_ *rand.Rand, min, max int) int {
-	return rand.Intn(max+1-min) + min
-}
-
-func createTempFileWithParams(f *testing.F) string {
-	file, err := os.CreateTemp("", "tmpParams-*.json")
-	require.NoError(f, err)
-	defer file.Close()
-	_, err = file.Write(paramsMarshalled)
-	require.NoError(f, err)
-	info, err := file.Stat()
-	require.NoError(f, err)
-	return filepath.Join(os.TempDir(), info.Name())
+func randRange(_ *rand.Rand, minV, maxV int) int {
```

Comment:
why is rand passed but not used?

Reply:
not sure, just moved the func 
I will modify to use it